### PR TITLE
Fix starting animals translation issue

### LINKED ISSUE #3: Starting animals don't match the UI language
Either localise the starting animals or delete them and add new questions to learn introduce the game and learn the first animal instead.

### PR DESCRIPTION
**Pull request by ezisazis**
resolves #3

### CODE REVIEW

**Comment by ezisazis** [comment]
@sinisterstuf afrikaans and swedish translation are still needed

**Comment by sionleroux** [comment]
I know technically the example first and second questions are about a dog and a big, but semantically they are example first and second questions. Wouldn't it be good to give the keys semantic names? Similarly to how the name of the app is `app_name`, not `guess_the_animal`.

File: app/src/main/res/values-sv/strings.xml [hunk 1]
```diff
@@ -23,4 +23,10 @@
     <string name="missing_name">Du har inte gett ditt djur ett namn!</string>
     <string name="missing_question">Du har inte skrivit en fråga!</string>
 
+    <string name="a_dog">Missing Translations</string>
+    <string name="dog_question">Missing Translations</string>
+
+    <string name="a_pig">Missing Translations</string>
+    <string name="pig_question">Missing Translations</string>
```
Comment (sionleroux) [inline app/src/main/res/values-sv/strings.xml:30]: The pig strings are actually the same as the ones for `learn_question`, I'll get back to you about the dog strings.

**Comment by ezisazis** [comment]
I like to do this way, because it's pointing to the meaning of the specific string. And probably more reusable than "example_first_question"

**Comment by sionleroux** [comment]
I don't see how `a_dog` is more reusable than `example_first_animal`. Would you like us to rename `missing_name` to `you_didnt_give_your_animal_a_name`? Similarly, how come you called the other string `pig_question` and not `does_it_have_a_curly_tail`? That's not even consistent.

Surely we can both agree that `pig_question` is more semantic than `does_it_have_a_curly_tail`, so why can't we agree that `a_pig` is not semantic? Even if it doesn't tell you the _words_ that are in the string, that's _the point_ you should be able to adjust and fix the words a bit without having to change the key.

For example, what happens if we decide to make the example first animal a cat and not a dog? If it was `a_dog` before you have to rename it to `a_cat` or alternatively leave it and let the value of `a_dog` be `a cat`, confusing developers (like api key vs access token in an API you and I both know and hate). Or if it's called something general that describes its _purpose_ instead of its _value_, e.g. `example_first_animal` and `example_first_question` or similar, then you can just update the value and carry on with life :smile:

I think my main point is: the key should not be the value otherwise they can't change independantly.

**Comment by ezisazis** [comment]
I think this whole example animal/question values will be purges from the strings and moved to a better place (eg.: Database), so I would not argue on these. `a_pig` / `a_dog` could be reusable as, later on you want to ask a question related to the current data that we have, the answers could be easily these strings. OTOH `dog_question` and `pig_questin` are badly named strings, but as I said hopefully they will be moved to another part of the application so I would not care about them.

**Comment by ezisazis** [comment]
I mean It's sure that the place of these values is not the strings.xml

**Comment by ezisazis** [comment]
I will fix the keys when you'll provide the translations.

**Comment by sionleroux** [comment]
Forgot to mention it in the commit, but this is unblocked by 3af6637, which removes the Swedish translations, and then I still need to write Afrikaans translations.

**Pull request closed without merging**